src/pages/workspaces/workspace/notebooks/modals/CloudEnvironmentModal.js

```diff
@@ -400,8 +395,7 @@ export const CloudEnvironmentModal = ({
     errorRuntimeId && h(RuntimeErrorModal, {
       runtime: _.find({ id: errorRuntimeId }, runtimes),
       onDismiss: () => setErrorRuntimeId(undefined)
-    }),
-    busy && spinnerOverlay
```

Comment:
Why do we want to remove these?

Reply:
Hmm.. I was trying to remove the layers of nested-blocking from this modal and the inner New(Runtime/Galaxy)Modals, but it seems that isn't possible since this modal can pause/stop. Essentially the behavior without this is, instead of blocking with a spinner when you click stop/start, it refreshes dynamically. This doesn't look the best when tested with simulated slow internet speed though... I'll add it back. Was kind of a rabbit hole to fix the issues I mention above so thanks for catching this :)